src/store.ts

```diff
@@ -52,65 +52,65 @@ const store: StoreOptions<RootState> = {
         },
     },
     actions: {
-        initKeys({ state, commit }) {
+        initWallets({ state, commit }) {
             // Fetch data from store
-            KeyStore.Instance.list().then((keyInfoEntries) => {
-                const keys = keyInfoEntries.map((keyInfoEntry) => KeyInfo.fromObject(keyInfoEntry));
-                commit('initKeys', keys);
+            WalletStore.Instance.list().then((walletInfoEntries) => {
+                const wallets = walletInfoEntries.map((walletInfoEntry) => WalletInfo.fromObject(walletInfoEntry));
+                commit('initWallets', wallets);
 
-                if (keys.length === 0) return;
+                if (wallets.length === 0) return;
 
                 // Try loading active
-                let activeKey: KeyInfo | undefined;
+                let activeWallet: WalletInfo | undefined;
                 let activeUserFriendlyAddress: string | null = null;
 
                 const storedRecentAccount = localStorage.getItem('_recentAccount');
                 if (storedRecentAccount) {
                     try {
                         const recentAccount = JSON.parse(storedRecentAccount);
-                        activeKey = state.keys.find((x) => x.id === recentAccount.loginId);
+                        activeWallet = state.wallets.find((x) => x.id === recentAccount.loginId);
                         activeUserFriendlyAddress = recentAccount.userFriendlyAddress;
```

Comment:
Could consider switching to "humanReadableAddress" (= non-issue!)

Reply:
This is a direct mapping to core's native `Address.toUserFriendlyAddress()`. It's also only internal to the Accounts Manager.